src/syntax_highlight/null.rs

```diff
@@ -0,0 +1,170 @@
+use liquid;
+use liquid::Token::{self, Identifier};
+use liquid::lexer::Element::{self, Expression, Tag, Raw};
+
+use pulldown_cmark as cmark;
+
+use error;
+
+pub fn has_syntax_theme(_name: &str) -> error::Result<bool> {
+    bail!("Themes are unsupported in this build.");
+}
+
+pub fn list_syntax_themes<'a>() -> Vec<&'a String> {
+    vec![]
+}
+
+pub fn list_syntaxes<'a>() -> Vec<String> {
+    vec![]
+}
+
```

Comment:
For a null "object", these are acting the most correct without them bailing.  No syntaxes and no themes are supported and they are accurately representing that.  `has_syntax_theme` is the one that is not acting as nicely but that is for the idea UX.